src/feedback/commands/submitFeedback.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { _Blob } from 'aws-sdk/clients/lambda'
+import _ = require('lodash')
+import * as vscode from 'vscode'
+import { ext } from '../../shared/extensionGlobals'
+import { ExtensionUtilities } from '../../shared/extensionUtilities'
+import { getLogger, Logger } from '../../shared/logger'
+import { BaseTemplates } from '../../shared/templates/baseTemplates'
+import { FeedbackTemplates } from '../templates/feedbackTemplates'
+import { submitFeedbackListener } from './submitFeedbackListener'
+
+export async function submitFeedback(listener?: (message: any) => Promise<void>) {
+    const logger: Logger = getLogger()
+
+    try {
+        const panel = vscode.window.createWebviewPanel('html', 'Submit Quick Feedback', vscode.ViewColumn.One, {
```

Comment:
You may want to make the panel a singleton; I'm pretty sure this iteration will let you have multiple instances of this panel open which doesn't make a ton of sense. I would opt for something like:
```
if (existing panel)
  focus on panel
else
  create panel and focus
```

Reply:
Right, I was going to look into this as well. Let me see if I can get this working.